src/src/facilitator/facilitator.service.ts

```diff
@@ -294,6 +294,61 @@ export class FacilitatorService {
 		}
 	}
 
+	async updateAadhaarDetails(id: number, body: any) {
+		const aadhaar_no = body.aadhar_no;
+
+		if (typeof aadhaar_no !== 'string' || !aadhaar_no.trim() || aadhaar_no.length !== 12) {
```

Comment:
add validation start with 2 and more not less then 2

Reply:
Done